### DISCUSSION TITLE
Displayclock command does not work with MAX7219

### DISCUSSION THREAD

**Discussion by Tinky1233** [arendst/Tasmota#21046] [Support request]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [x] Self-compiled
- [ ] Flashing tools used: _____
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
#ifdef CODE_IMAGE_STR
  #undef CODE_IMAGE_STR
#endif
#define CODE_IMAGE_STR "MatrixDisplay"
#ifndef USE_DISPLAY
#define USE_DISPLAY
#endif

#define USE_DISPLAY_MAX7219_MATRIX

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
Displaymodel 19
Display Mode 0
Power 1
DisplayDimmer 100

DisplayClock 2

### EXPECTED BEHAVIOUR
Time on display

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
It stoped working after version 11.0 , DisplayText works

**Comment by Jason2866** [comment]
please add to your `user_config_override.h`
`#define USE_DISPLAY_MODES1TO5`

**Comment by Tinky1233** [comment]
Yes, it now works! Thank you!

This should be mentioned in manual:
https://tasmota.github.io/docs/MAX7219/
https://github.com/arendst/Tasmota/discussions/13590
